src/command_modules/azure-cli-batch/HISTORY.rst

```diff
@@ -5,9 +5,9 @@ Release History
 
 3.1.6
 +++++
+* Fixed bug in pool create command when a resource ID was used with the --image flag.
```

Comment:
We have cut a release before this PR, please add this history to the `3.1.7` section.

Reply:
Please also bump the version in the setup.py to `3.1.7`